### ISSUE TITLE
Please add support for Windows as OS

### ISSUE THREAD

**Issue by ntimo** [grafana/pyroscope-dotnet#49]
Currently the profiler only supports running Dotnet applications on Linux but being able to also profile them while running on windows would be quite nice, since a lot of people probably run dotnet apps on Windows using IIS.

**Comment by mhascak** [comment]
+1

**Comment by tkononowiczsolwit** [comment]
+1

**Comment by yahyaozturk** [comment]
+1

**Comment by gmkado** [comment]
I want to add pyroscope to windows but [the docs](https://grafana.com/docs/pyroscope/latest/configure-client/language-sdks/dotnet/#configure-the-dotnet-client) only show how to configure for linux.

Investigating these repos, I'm trying to understand how this issue relates to [this one](https://github.com/grafana/pyroscope/issues/166) in the main pyroscope repo.  Is there a way to compile pyroscope for windows and its just not currently distributed?

**Comment by giard-alexandre** [comment]
Yes! This feels like a MASSIVE missing feature. Profiling desktop apps on windows would be great.

**Comment by stebet** [comment]
I'd love to see Windows support as well. I can't really see myself using Pyroscope or recommending to others really until it has Windows support as well. There must be a way that it can be used to plug into the [.NET profiling APIs on Windows](https://learn.microsoft.com/en-us/dotnet/core/unmanaged-api/profiling/) as well.

**Comment by aleks-p** [comment]
this has been reevaluated, we now plan to provide experimental Windows support

**Comment by aleks-p** [comment]
This is now supported with the [v1.3.0](https://github.com/grafana/pyroscope-dotnet/releases/tag/pyroscope-1.3.0) version of the profiler. The usage is identical to the Linux profiler but we plan to add an example based on the existing [rideshare](https://github.com/grafana/pyroscope/tree/main/examples/language-sdk-instrumentation/dotnet/rideshare) app in the coming days/weeks.

See the [docs](https://grafana.com/docs/pyroscope/latest/configure-client/language-sdks/dotnet/#compatibility) for info on compatibility. Note that the compatibility matrix (cpu architectures / .NET versions) is not set in stone and can change, we are starting small and can expand from here when possible.

Also note that the support is in [public preview](https://grafana.com/docs/release-life-cycle/). There could be issues and it might take us time to address them - take this into account if you plan to use this in production.
